### ISSUE TITLE
[Bug]: Viper's Rattling Coils and Anguine Tribute no longer respect Hide When Inactive toggle

### ISSUE THREAD

**Issue by EmmettJayhart** [DelvUI/DelvUI#1229]
### What are you trying to do?

Track Rattling Coils in Viper's Job Gauge

### What is the expected behavior?

Job Gauge should be showing an empty Rattling Coils when Hide When Inactive is toggled off

### What actually happened?

Job Gauge is hiding the empty Rattling Coils irregardless of the Hide When Inactive toggle

### Suggested solution

Rattling Coils is in the right place
![image](https://github.com/user-attachments/assets/7092f736-8178-4765-bb7d-8ee53387db18)

Default Profile also has this issue
![image](https://github.com/user-attachments/assets/8cc66565-ae0c-4282-868e-f8755c6e0e87)

Restarting did not fix this

### Logs

```shell
Dalamud Log window on Verbose did not show any relevent logs
```

### FFXIV Update

- [X] I have confirmed that I have the latest version of XIVLauncher and DelvUI.

**Comment by EmmettJayhart** [comment]
I found the culprit.
Commit 96cf67a refactored this section, but the inversion of the logic was incomplete.
![image](https://github.com/user-attachments/assets/74235563-729c-42cf-b08e-c0491271aac8)

It looks like the same issue exists for Anguine Tribute.

**Comment by Tischel** [comment]
Fixed in v2.2.0.6
